Sources/Models/ConversationUpdate.swift

```diff
@@ -5,12 +5,62 @@ public struct Message: Codable {
         case user = "user"
         case assistant = "assistant"
         case system = "system"
+        case tool = "tool"
+        // CHANGE: Added unknown case to handle unexpected/missing role values
+        // WHY: Prevents decoding failures if Vapi adds new role types or omits the role field.
+        // All invalid/missing roles default to .unknown instead of failing the entire message
+        case unknown = "unknown"
     }
     
+    // CHANGE: Role is non-optional but defaults to .unknown for invalid/missing values
+    // WHY: Simpler for consumers (no need to unwrap optionals) while still handling
+    // missing or unrecognized roles gracefully
     public let role: Role
-    public let content: String
+    public let content: String?
+    // CHANGE: Added toolCalls field to parse function calls from conversation history
+    // WHY: Vapi includes tool_calls in conversation-update messages for tracking assistant function invocations
+    public let toolCalls: [ToolCallItem]?
+    public let toolCallId: String?
```

Comment:
❓ I see that there can be multiple tool calls (i.e. the `[ToolCallItem]` array). However, there's a single tool call id. What tool call does this id relate to?